__tests__/app.test.js

```diff
@@ -59,3 +59,43 @@ describe('GET /api/', () =>{
     })
 });
 
+//4
+describe('GET /api/articles/:article_id', () => {
+    test('responds with status 200 and with article object', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({body}) => {    
+      
+            expect(body.article.article_id).toBe(1)
+            expect(body.article.title).toBe("Living in the shadow of a great man" )
+            expect(body.article.topic).toBe("mitch")
+            expect(body.article.author).toBe("butter_bridge")
+            expect(body.article.body).toBe("I find this existence challenging")
+            expect(body.article.created_at).toBe('2020-07-09T20:11:00.000Z' )
+            expect(body.article.votes).toBe(100)
+            expect(body.article.article_img_url).toBe('https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700')
```

Comment:
This works fine and doesn't need changing.
However going forward you may want to look into other Jest matchers available to you.
ObjectContaining and ToMatchObject are two that work very well for testing these responses.